instructure/instructure-ui
# chore: move commitizen config to commitlint;

also removed cz-lerna-changelog pkg and skip commit hook on rebase and amend

`update:package:list` and `commit:package:list` was removed from package.json as I wasn't able to figure out why they were needed (probably legacy usecase)

the package `instui-config` was removed too as it isn't serving any purpose anymore

test plan:
- check out the branch and make any change and commit it -> check if the prompt works as intended
- make a change 2 different packages -> check if scopes are correctly calculated
- in the scope select menu, select more then 3 packages -> the final messages should display `many` as the scope
- char limit should be 150 -> after the prompt, extend the message over the limit -> commitlint should give an error
- amend a commit -> prompt should not appear
- rebase the branch -> prompt should not appear
